examples/gpu_benchmark.ipynb

```diff
@@ -34,12 +34,12 @@
     "import numpy as np\n",
     "from time import time \n",
     "\n",
-    "from fastai import *\n",
-    "from fastai.basic_data import *\n",
-    "from fastai.basic_data import *\n",
-    "from fastai.tabular import *\n",
-    "from fastai.basic_data import DataBunch\n",
-    "from fastai.tabular import TabularModel\n",
+    "import fastai\n",
+    "from fastai.basics import Learner\n",
+    "from fastai.tabular.model import TabularModel\n",
+    "from fastai.tabular.data import TabularDataLoaders\n",
```

Comment:
Thanks for changing from wildcard imports ```from fastai import *```!  using import * isn't allowed by flake8 on our python files - we should probably be adding something to run flake8/black on our notebooks as well